### ISSUE TITLE
Python 3.11 not supported

### ISSUE THREAD

**Issue by BlockyTheDev** [py2exe/py2exe#155]

**Comment by albertosottile** [comment]
WIP, should be ready soon. Preliminary wheels available here: https://github.com/py2exe/py2exe/actions/runs/3332827887

**Comment by BlockyTheDev** [comment]
Thanks for the update to the support state.

**Comment by bersbersbers** [comment]
I installed from https://github.com/py2exe/py2exe/suites/8980859163/artifacts/413204060 (unzip, then `pip install`).

This is what I get:

```
(project_dev) C:\Code\project\project>python -m py2exe project.py
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\bers\.pyenv-win-venv\envs\project_dev\Lib\site-packages\py2exe\__main__.py", line 8, in <module>
    build_exe.main()
  File "C:\Users\bers\.pyenv-win-venv\envs\project_dev\Lib\site-packages\py2exe\build_exe.py", line 136, in main
    builder = runtime.Runtime(options)
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\bers\.pyenv-win-venv\envs\project_dev\Lib\site-packages\py2exe\runtime.py", line 159, in __init__
    self.options.dll_excludes = self.options.dll_excludes if self.options.dll_excludes else []
                                                             ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Namespace' object has no attribute 'dll_excludes'
```

**Comment by albertosottile** [comment]
@bersbersbers In general the `py2exe` command is not supported and will be actually removed from 0.13 (see https://github.com/py2exe/py2exe/blob/master/README.md). Please refer to the `freeze` API documentation https://github.com/py2exe/py2exe/blob/master/docs/py2exe.freeze.md and to #150 for further information.

**Comment by bersbersbers** [comment]
@albertosottile thanks! Does this mean you will still remove the output from `python -m py2exe`, which currently prints
> usage: __main__.py [-h] [-i modname] [-x modname] [-p package_name] [-O] [-s] [-r] [-f modname] [-v] [-c] [-d DESTDIR] [-b {0,1,2,3}] [-W setup_path] [-svc service] [script ...]
__main__.py: error: nothing to build

**Comment by albertosottile** [comment]
@bersbersbers Indeed, also this CLI will be removed in 0.13.0.0

**Comment by albertosottile** [comment]
Fixed in 0.13.0.0
